<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLanguage</class>
 <widget class="QDialog" name="DialogLanguage">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Language</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/Resource/Generic/Language.png</normaloff>:/Resource/Generic/Language.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frameCommonHeader">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Resource/HeaderBackground.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_23">
      <property name="margin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelLogo">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resource.qrc">:/Resource/Quazaa48.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelHeadertext">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;Quazaa!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="listWidgetLanguages">
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>English (Default)</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-en.png</normaloff>:/Resource/Languages/default-en.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Afrikanns</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-af.png</normaloff>:/Resource/Languages/default-af.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Arabic العربية</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-ar.png</normaloff>:/Resource/Languages/default-ar.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Català</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-ca.png</normaloff>:/Resource/Languages/default-ca.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>简体中文</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-chs.png</normaloff>:/Resource/Languages/default-chs.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Čeština</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-cz.png</normaloff>:/Resource/Languages/default-cz.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Deutsch</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-de.png</normaloff>:/Resource/Languages/default-de.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Eesti</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-ee.png</normaloff>:/Resource/Languages/default-ee.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Español</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-es.png</normaloff>:/Resource/Languages/default-es.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Español Mexicano</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-es-mx.png</normaloff>:/Resource/Languages/default-es-mx.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Suomi</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-fi.png</normaloff>:/Resource/Languages/default-fi.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Français</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-fr.png</normaloff>:/Resource/Languages/default-fr.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ελληνικά</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-gr.png</normaloff>:/Resource/Languages/default-gr.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>עברית</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-heb.png</normaloff>:/Resource/Languages/default-heb.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hrvatski</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-sr.png</normaloff>:/Resource/Languages/default-sr.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Magyar</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-hu.png</normaloff>:/Resource/Languages/default-hu.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Italian</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-it.png</normaloff>:/Resource/Languages/default-it.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>日本語</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-ja.png</normaloff>:/Resource/Languages/default-ja.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lietuvių</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-lt.png</normaloff>:/Resource/Languages/default-lt.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nederlands</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-nl.png</normaloff>:/Resource/Languages/default-nl.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Norsk</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-no.png</normaloff>:/Resource/Languages/default-no.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Polski</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-pl.png</normaloff>:/Resource/Languages/default-pl.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Português Brasileiro</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-pt-br.png</normaloff>:/Resource/Languages/default-pt-br.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pусский</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-ru.png</normaloff>:/Resource/Languages/default-ru.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slovenščina</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-sl-si.png</normaloff>:/Resource/Languages/default-sl-si.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shqip</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-sq.png</normaloff>:/Resource/Languages/default-sq.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Srpski</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-hr.png</normaloff>:/Resource/Languages/default-hr.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Svenska</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-sv.png</normaloff>:/Resource/Languages/default-sv.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Türkçe</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-tr.png</normaloff>:/Resource/Languages/default-tr.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>繁體中文</string>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Languages/default-tw.png</normaloff>:/Resource/Languages/default-tw.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayoutFooter">
     <property name="margin">
      <number>4</number>
     </property>
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <spacer name="horizontalSpacerFooter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonOK">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
